docs/section6/Rule6-1.md

```diff
@@ -22,13 +23,15 @@
 
   - If building segment specifies lighting building area type, get the allowable lighting power density from Table G3-8: `if building_segment_p.lighting_building_area_type in table_G3_8: allowable_LPD_BAM = data_lookup(table_G3_8, building_segment_p.lighting_building_area_type)`  
 
-  - For each thermal block in building segment: `thermal_block_p in building_segment_p.thermal_blocks:`  
+    - For each zone in thermal block: `zone_p in building_segment_p.zones:`  
```

Comment:
Suggest removing the term thermal block. I don't think this in the schema at all.

Reply:
maybe just replace it with building segment?